misc/python/materialize/mzcompose/__init__.py

```diff
@@ -58,6 +58,7 @@
     # Others (ordered by name)
     "cluster_always_use_disk": "true",
     "compute_dataflow_max_inflight_bytes": "134217728",  # 128 MiB
+    "compute_hydration_concurrency": 4,
```

Comment:
optional: I'd suggest 2, so we exercise this more

Reply:
Leaving it at 4 for now. Some of the nightlies don't pass with 2. I know why and I have a fix (#26016) but it's a persist_sink change so I'd like to wait for Aljoscha when he's back next week.